src/plugins/discover/public/components/discover_grid/discover_grid.scss

```diff
@@ -57,6 +32,29 @@
   flex-direction: column;
   flex-wrap: nowrap;
   height: 100%;
+
+  .euiDataGrid__controls {
+    border: none;
+    border-bottom: $euiBorderThin;
+  }
+
+  .euiDataGridRowCell.euiDataGridRowCell--controlColumn[data-gridcell-column-id='select'] {
+    padding: $euiSizeS 0 0 $euiSizeS;
+  }
+
+  .euiDataGridRowCell.euiDataGridRowCell--controlColumn[data-gridcell-column-id='openDetails'] {
+    padding: $euiSizeXS 0 0 $euiSizeXS;
+  }
+
+  .dscDiscoverGrid__table .euiDataGridRowCell:first-of-type {
+    border-left: none;
+    border-right: none;
+  }
```

Comment:
Out of curiosity, is this styling necessary? If it's to prevent the checkbox from being cut-off by 2px, would it be better to just change the width of the column to account for the additional 2px of transparent border, rather than overriding EUI styles? Or are the transparent borders an EUI bug that needs to be addressed?

Reply:
Thanks for the idea, @MichaelMarcialis! 
Updated via https://github.com/elastic/kibana/pull/164187/commits/de5375709c97b34dacae417c53d3b53ec4e28175